plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergOrcSystemTables.java

```diff
@@ -22,4 +27,16 @@ public TestIcebergOrcSystemTables()
     {
         super(ORC);
     }
+
+    @Test
+    public void testPropertiesTable()
+    {
+        try (TestTable table = newTrinoTable("test_properties_table", "AS SELECT 1 x")) {
+            assertThat(getTableProperties(table.getName()))
+                    .contains(
+                            entry("write.format.default", "ORC"),
+                            entry("write.orc.compression-codec", "zstd"),
+                            entry("write.parquet.compression-codec", "zstd"));
```

Comment:
let's maybe add a link to the issue pointed out by @ebyhr to explain why this seemingly unrelated property is present here.